lib/everypolitician_extensions.rb

```diff
@@ -38,8 +38,27 @@ def memberships_at_end
       end
     end
   end
+
+  module MembershipExtension
+    def group
+      popolo.organizations.find_by(id: on_behalf_of_id)
+    end
+
+    def area
+      popolo.areas.find_by(id: area_id)
+    end
+
+    def group_name
+      group.name.to_s.downcase == 'unknown' ? '' : group.name
+    end
+
+    def area_name
+      area.name if area_id
+    end
+  end
 end
```

Comment:
I'm a little hesitant about this one, too. Are we going to create equivalent methods like this for every other method that someone might call on an Area? If we really need something like this, then I suspect we want `def area` to return some sort of NullArea if none exists.